packages/sodium/src/js/modules/Utils.js

```diff
@@ -11,3 +11,16 @@ const sizeFormat = (bytes, exponent) => {
 export const megaSizeFormat = bytes => sizeFormat(bytes, 2);
 
 export const kiloSizeFormat = bytes => sizeFormat(bytes, 1);
+
+export const jsonParseSafe = (text, defaultValue = {}) => {
+  try {
+    const value = JSON.parse(text);
+    // undefinedとnullは存在しないプロパティにアクセスすると
+    // エラーを投げるので、代わりにdefaultValueを返す
+    return value === undefined || value === null
+      ? defaultValue
+      : value;
```

Comment:
[Null 合体](https://developer.mozilla.org/ja/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) を使うならば

```suggestion
    return value ?? defaultValue;
```

Reply:
```
ERROR in ./src/js/modules/Utils.js 20:18
Module parse failed: Unexpected token (20:18)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|     // undefinedとnullは存在しないプロパティにアクセスすると
|     // エラーを投げるので、代わりにdefaultValueを返す
>     return value ?? defaultValue;
|   } catch(e) {
|     return defaultValue;
 @ ./src/js/App.js 3:0-48 41:36-49 42:37-50 43:40-53 44:40-53 45:45-58 65:52-65
```
ビルドエラーになりますね。。
`@babel/plugin-proposal-nullish-coalescing-operator` を使うと通るような情報もあるんですが、どう読み込ませるのか不明です。
.babelrc.jsonやpackage.jsonにbabelの設定を書く方法で解決できず。

Reply:
> `@babel/plugin-proposal-nullish-coalescing-operator` を使うと通るような情報もあるんですが、どう読み込ませるのか不明です。

なるほど。おそらくそもそもbabelが存在しないのではないでしょうか。
導入されている想定でしたが無いということならビルドに失敗するだけなのでスルーしてください。
ちなみに導入するのであれば、[`babel-loader @babel/core @babel/preset-env` あたりインストールして設定する必要ある](https://webpack.js.org/loaders/babel-loader/)かと思います。
`@babel/plugin-proposal-nullish-coalescing-operator` は通常 `@babel/preset-env` によって提供されるかなと思います。

Reply:
ビルド構成とビルド結果が変わりそうなので、今回はこのままでいこうと思います。